### PR TITLE
[Feature][Feature/AES-1878]: FooterBlock Component for New Footer

### PR DESCRIPTION
**Pull request by zack76**
https://aesoponline.atlassian.net/browse/AES-1523

Screen Recording 2020-09-16 at 3 29 48 pm
Screen Recording 2020-09-16 at 3 30 24 pm

### CODE REVIEW

File: src/components/FooterBlock/FooterBlock.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    styles.base,
+    styles[theme],
+    links && !isVisibleOnTabletAndMobile && styles.invisible,
+    className,
+  );
+  const titleClassSet = cx(styles.title);
+  const dividerClassSet = cx(styles.divider, styles[theme]);
+  return (
+    <section className={baseClassSet}>
+      <div className={titleClassSet}>{title}</div>
```
Comment (kevin-ho87) [inline src/components/FooterBlock/FooterBlock.js:31]: Are you able to use a heading eg: `<h5 />` for the title/heading. For html semantics/accessibility.

File: src/components/FooterBlock/FooterBlock.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  copy,
+  isVisibleOnTabletAndMobile,
+  links,
+  theme,
+  title,
+}) => {
+  const baseClassSet = cx(
+    styles.base,
+    styles[theme],
+    links && !isVisibleOnTabletAndMobile && styles.invisible,
```
Comment (kevin-ho87) [inline src/components/FooterBlock/FooterBlock.js:24]: I was wondering if you could use the `<Hidden />` component to control the display logic. Just a thought.

File: src/components/FooterBlock/FooterBlock.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                  : INTERNAL_NO_ICON_TEXT_LINK
+              }
+              theme={theme}
+              title={link.text}
+              url={link.url}
+            >
+              {link.text}
+            </Hyperlink>
+          </div>
+        ))}
```
Comment (kevin-ho87) [inline src/components/FooterBlock/FooterBlock.js:52]: Are you able to update this to a list structure `ul > li* > a` from html accessibility.

File: src/components/FooterBlock/FooterBlock.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  );
+};
+
+FooterBlock.propTypes = {
+  className: PropTypes.string,
+  copy: PropTypes.string,
+  isVisibleOnTabletAndMobile: PropTypes.bool,
+  links: PropTypes.array,
+  theme: PropTypes.oneOf(['dark', 'light']),
+  title: PropTypes.string,
```
Comment (kevin-ho87) [inline src/components/FooterBlock/FooterBlock.js:64]: Could you change `title` to `heading`. Just that `title` is a reserve html keyword.

**Review by kevin-ho87 — CHANGES_REQUESTED** [review]

File: src/components/FooterBlock/FooterBlock.js (hunk 1 above)
Comment (beau-charman) [reply src/components/FooterBlock/FooterBlock.js:31]: <Heading level="5" />

File: src/components/FooterBlock/FooterBlock.js (hunk 4 above)
Comment (beau-charman) [reply src/components/FooterBlock/FooterBlock.js:64]: Thank you @kevin-ho87!

**Review by beau-charman — APPROVED** [review]

**Comment by beau-charman** [comment]
Just a few things to address

File: src/components/FooterBlock/FooterBlock.js (hunk 1 above)
Comment (zack76) [reply src/components/FooterBlock/FooterBlock.js]: updated

File: src/components/FooterBlock/FooterBlock.js (hunk 2 above)
Comment (zack76) [reply src/components/FooterBlock/FooterBlock.js]: updated

File: src/components/FooterBlock/FooterBlock.js (hunk 3 above)
Comment (zack76) [reply src/components/FooterBlock/FooterBlock.js]: updated

File: src/components/FooterBlock/FooterBlock.js (hunk 4 above)
Comment (zack76) [reply src/components/FooterBlock/FooterBlock.js]: updated

**Review by kevin-ho87 — APPROVED** [review]

**Pull request merged**